core/tests/protractor/collections.js

```diff
@@ -83,6 +81,32 @@ describe('Collections', function() {
     users.logout();
   });
 
+  it('interacts with the collection player as a guest', function() {
+    collectionPlayer.goToDemoCollection();
+    // Check that we are in the guest state (sign in button visible).
+    collectionPlayer.verifySignInButton();
+    // Verify that there are 5 uncompleted explorations (initial state).
+    collectionPlayer.verifyNumUncompletedExplorations(5);
+    // Play the third exploration in the demo collection.
+    collectionPlayer.playDemoExplorationThree();
```

Comment:
The problem with this is that playing a specific exploration isn't something that's really specific to the collection player. I'd put the actions to go through the exploration here (in the test itself) -- see also my other comment about creating your own collection whose explorations are "simple" and easily completeable.